format.go

```diff
@@ -73,13 +75,14 @@ func ToString(err error, withTrace bool) string {
 func ToCustomString(err error, format StringFormat) string {
 	upErr := Unpack(err)
 
-	// return early if this is a third-party error
-	if upErr.ExternalErr != "" {
-		return fmt.Sprint(upErr.ExternalErr)
-	}
-
 	var str string
 	if format.Options.InvertOutput {
+		if format.Options.WithExternal && upErr.ErrExternal != nil {
```

Comment:
since this is an if/else situation, i'm not sure i understand where the re-use comes in. i can try to pull one or two things into a common location though

Reply:
reuse of common variables derived after parsing logic

Reply:
i tried this and it ended up being even more verbose so i'm planning to leave this as is